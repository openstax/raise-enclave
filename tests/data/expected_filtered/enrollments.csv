user_uuid,course_id,role
d35a986b-a7ad-4c61-8e76-205eccaf9a06,3,student