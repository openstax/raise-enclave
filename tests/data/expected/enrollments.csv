user_uuid,course_id,role
7080c78d-298b-40ba-a68d-55d6a93b00fb,2,student
7080c78d-298b-40ba-a68d-55d6a93b00fb,2,student
7080c78d-298b-40ba-a68d-55d6a93b00fb,2,teacher
7080c78d-298b-40ba-a68d-55d6a93b00fb,2,student
c6a4a01e-b112-475f-82ff-b051ab1d44e1,2,student
